### DISCUSSION TITLE
Percentages with or without decimals?

### DISCUSSION THREAD

**Discussion by SanderDevisscher** [inbo/reporting-rshiny-grofwildjacht#274] [Q&A] [answered]
@jimcasaer Currently some percentages below graphs/maps are displayed without decimals, while others have 1 (see examples below). I think it is best to pick one and apply it to all. I would like to advocate in favor of displaying the percentages with 1 decimal instead of none. Especially when the percentage rounds up to 100% this can be confusing see example below.

without decimal
![image](https://user-images.githubusercontent.com/16779320/154672790-8e76be53-9cc6-40fe-a1dc-9506e98e3f89.png)
This is nearly 100% but not exactly, its 99.5273....%. In this case I think 99.5% would better cover reality. 

with decimal
![image](https://user-images.githubusercontent.com/16779320/154672877-1790eb5a-d690-46af-b2b2-53a15741a246.png)
Such clearity :-p

**Comment by jimcasaer** [comment, marked as answer]
1 decimal looks perfect to me ! and yes : all over the page the same number of decimals for percentages
